documentation/tutorials/get-started-with-ash-and-phoenix.md

```diff
@@ -0,0 +1,505 @@
+
+## Requirements
+
+If you want to follow along yourself, you will need the following things:
+
+1. Elixir (1.12 or later) and Erlang (22 or later) installed
+2. PostgreSQL installed
+3. A text editor
+4. A terminal to run the examples
+
+## Setup
+
+### Create a New Phoenix Project
+
+_This section is based on the [Phoenix installation docs](https://hexdocs.pm/phoenix/installation.html). For more details go there._
+
+First we need to install the Phoenix project generator, then we'll run the generator to create our new project.
+
+**NOTE: DO NOT run `mix ecto.create`, (as it asks you to) we will do this the Ash way later.**
+
+```bash
+# install Phoenix project generator
+$ mix archive.install hex phx_new
+
+# generate Phoenix project
+$ mix phx.new my_ash_phoenix_app
+
+# cd into project
+$ cd my_ash_phoenix_app
+```
+
+### Add Dependencies
+
+We now need to add Ash, AshPhoenix and AshPostgres to our Phoenix project. We need to add the dependencies to the `deps` function in our `mix.exs`. We'll also need to add dependencies to our `.formatter.exs` to ensure consistent formatting when using `mix format`.
+
+```elixir
+# mix.exs
+
+  def deps do
+    [
+      {:phoenix, "~> 1.6.15"},
+      # ...
+      {:ash, "~> 2.6.2"},
+      {:ash_postgres, "~> 1.3.10"},
+      {:ash_phoenix, "~> 1.2.5"}
+    ]
+  end
+```
+
+```elixir
+# .formatter.exs
+[
+  import_deps: [:ecto, :phoenix, :ash, :ash_phoenix, :ash_postgres],
+  inputs: ["*.{ex,exs}", "priv/*/seeds.exs", "{config,lib,test}/**/*.{ex,exs}"],
+  subdirectories: ["priv/*/migrations"]
+]
+```
+
+Now in the terminal install these new dependencies.
+
+```bash
+$ mix deps.get
+```
+
+### Use `AshPostgres.Repo` and Create the Database
+
+We need to swap `Ecto.Repo` for `AshPostgres.Repo`. `AshPostgres.Repo` enriches your repo with additional AshPostgres specific behaviour, but is essentially a thin wrapper around `Ecto.Repo`.
+To use `AshPostgres.Repo` change your repo module to look like this:
+
+```elixir
+# lib/my_ash_phoenix_app/repo.ex
+
+defmodule MyAshPhoenixApp.Repo do
+  use AshPostgres.Repo, otp_app: :my_ash_phoenix_app
+
+  # Installs Postgres extensions that ash commonly uses
+  def installed_extensions do
+    ["uuid-ossp", "citext"]
+  end
+end
+```
+
+After swapping `AshPostgres.Repo` in, you can now create the database using:
+
+```
+$ mix ash_postgres.create
+
+The database for MyAshPhoenixApp.Repo has been created
+```
+
+### Edit Config
+
+We need to specify the Ash APIs that our application uses and some config for backwards compatibility that will be removed in the next major release.
+Add this to your config:
+
+```elixir
+# config/config.exs
+
+import Config
+
+# For backwards compatibility, the following configuration is required.
+# see https://ash-hq.org/docs/guides/ash/latest/get-started#temporary-config for more details
+config :ash, :use_all_identities_in_manage_relationship?, false
+
+config :my_ash_phoenix_app,
+  ash_apis: [MyAshPhoenixApp.Blog]
```

Comment:
Does this part replace the `config :my_app, ecto_repos: [MyApp.Repo]` line? Or should be in addition to it?

Reply:
It doesn't replace `ecto_repos` it is in addition to it. At the top of the code block I say: "Add this to your config:"